src/zcert.c

```diff
@@ -98,6 +98,34 @@ zcert_new_from (const byte *public_key, const byte *secret_key)
     return self;
 }
 
+#ifdef CZMQ_BUILD_DRAFT_API
```

Comment:
you don't need to wrap the implementation in the ifdef - it's exported as private by default